src/main/java/io/confluent/idesidecar/restapi/util/OperatingSystemType.java

```diff
@@ -24,6 +25,10 @@ public static OperatingSystemType from(Properties system) {
     if (lowerName.contains("mac") || lowerName.contains("darwin")) {
       return OperatingSystemType.MacOS;
     }
+    if (lowerName.contains("unix" ) || lowerName.contains("sunos") || lowerName.contains("solaris")
```

Comment:
nit: Let's remove this trailing whitespace.
```suggestion
    if (lowerName.contains("unix") || lowerName.contains("sunos") || lowerName.contains("solaris")
```